src/libpmem2/pmem2_windows.c

```diff
@@ -35,71 +35,109 @@
  */
 
 #include "libpmem2.h"
+#include "out.h"
 #include "pmem2.h"
 #include "pmem2_utils.h"
 
+#define PMEM2_HIDWORD(x) ((DWORD)((x) >> 32))
+#define PMEM2_LODWORD(x) ((DWORD)((x) & 0xFFFFFFFF))
+
 /*
- * pmem2_map -- XXX
+ * mapping -- creates file mapping object for a specified file
  */
-int
-pmem2_map(struct pmem2_config *cfg, struct pmem2_map **mapp)
+HANDLE
+mapping(const struct pmem2_config *cfg, DWORD protect, int *err)
```

Comment:
I guess it is not common in our codebase to use a noun as a function name. I suggest a verb + noun combination e.g. create_mapping.